factory/networkComponents.go

```diff
@@ -168,7 +168,7 @@ func (ncf *networkComponentsFactory) Create() (*networkComponents, error) {
 		return nil, err
 	}
 
-	ncf.waitForBootstrap(ncf.bootstrapWaitSeconds)
+	netMessenger.WaitForConnections(time.Duration(ncf.bootstrapWaitSeconds)*time.Second, ncf.p2pConfig.Node.WaitNumPeersOnBootstrap)
```

Comment:
[optional]
If you find it suitable, perhaps extract this `time.Duration(ncf.bootstrapWaitSeconds)*time.Second` in a new variable (or declare a const time var/ or store it inside struct as time var field and initialize it in constructor)

Reply:
done